src/App.js

```diff
@@ -9,94 +9,97 @@ class App extends Component {
   constructor() {
     super();
     this.state = {
-      areGamesLoaded: false,
```

Comment:
instead areGamesLoaded and areGenresLoaded, we are able to check against there being a value in games/genre respectively.

Reply:
Great idea. This definitely makes our app easier to read and more concise.